lib/mayaUsd/ufe/UsdUndoDeleteCommand.h

```diff
@@ -53,7 +53,8 @@ class MAYAUSD_CORE_PUBLIC UsdUndoDeleteCommand : public Ufe::UndoableCommand
     void redo() override;
 
 private:
-    PXR_NS::UsdPrim _prim;
+    PXR_NS::UsdPrim         _prim;
+    PXR_NS::UsdStageWeakPtr _stage;
```

Comment:
You need this stage in the `executeCommand`. There is no reason that I can find to store it as a member of the class. Simply instantiate it from prim (prim.GetStage()) prior to any delete operation.